streamer/src/nonblocking/quic.rs

```diff
@@ -128,9 +135,11 @@ pub fn spawn_server(
     max_streams_per_ms: u64,
     wait_for_chunk_timeout: Duration,
     coalesce: Duration,
-) -> Result<(Endpoint, Arc<StreamStats>, JoinHandle<()>), QuicServerError> {
+) -> Result<SpawnNonBlockingServerResult, QuicServerError> {
     info!("Start {name} quic server on {sock:?}");
-    let (config, _cert) = configure_server(keypair, gossip_host)?;
+    let concurrent_connections = max_staked_connections + max_unstaked_connections;
+    let max_concurrent_connections = concurrent_connections + concurrent_connections / 4;
```

Comment:
ditto comment from 1.17 bp. consider endpoint count?

Reply:
That need to be handled we have multi quic -- this only has one endpoint

Reply:
oh do we not have the single endpoint version of multiquic in? thought that landed last week :thinking: